_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+</figure>
+
+No matter how you shake it, though, land measurements in the West are consistently divisible by the magic 66 foot Gunter Chain. You’ll see its effect everywhere. This measuring unit was so successful because it made the math easy - and many early surveyors had a shaky grasp of mathematics and they were required to work quickly.
+
+These early surveyors typically placed a permanent monument at Section and Quarter-Section corners. These markers are still heavily used today and are the starting point for almost every Western legal description you’ll encounter - including land deeds (the transfer of private property), water rights (well locations), taxing areas, municipal annexations - and consequently voting districts. Because of this, it has been difficult for the US to adopt the International System of Units (the modern metric system).
+
+It is precisely this survey pattern that explains why the Western United States is so orthogonal. Although the grid is not unique to the West, it's certainly at the core of all land sales and as a result it influences many development patterns. Let’s look at a few examples.
+
+## The Western Grid
+{: .text-left}
+
+<figure class="caption caption--left caption--flow">
+  <a href="{{ "/images/plat_of_zion.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/plat_of_zion_sm.png" | prepend: site.baseurl }}" class="width: 100%;" alt="Plat of Zion">
+  </a>
+  <figcaption class="caption__text">Fig 7: Plat of Zion</figcaption>
+</figure>
+<figure class="caption caption--right caption--flow">
+  <a href="{{ "/images/ColonialBoston.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/ColonialBoston_sm.png" | prepend: site.baseurl }}" alt="Colonial Boston">
+  </a>
+  <figcaption class="caption__text">Fig 8: Colonial Boston</figcaption>
+</figure>
+
+One of the more dominant patterns you'll see in the West is the rectangular city block. You could argue that the Mormon pioneers helped carry this development pattern west from Missouri with their Plat of Zion (Fig 7) - and their subsequent establishment of more than 700 communities. Contrast this pattern with the pre-PLSS pattern of the early colonial settlers in Boston (Fig 8).
+
+Large city blocks are a common theme in the West. These typically vary between 198 feet, 330 feet, and 660 feet (which is 3, 5, and 10 chains). Often, two sides of the block are 660 feet, with the adjacent sides being 198 feet or 330 feet. Looking at Portland (Fig 9), their blocks are unusually small in the West at 198 feet x 198 feet. On the other side of the spectrum, Salt Lake City (Fig 9) is known for having the largest blocks in the West at 660 feet x 660 feet. Blocks are more rectangular in San Francisco (Fig 9) and Phoenix (Fig 9), as they measure 660 feet x 198 feet and 660 feet x 330 feet, respectfully.
```

Comment:
Footnote 9 (For Figure 9): 
Greg Bunce, “Variations on the Western Urban Grid,” Automated Geographic Reference Center, March 01, 2019.